tests/test_callprofiler.py

```diff
@@ -192,3 +196,90 @@ def foo():
 
         modmesh.call_profiler.reset()
         foo()
+
+    def test_get_stat(self):
+        time1 = 0.5
+        time2 = 0.1
+        time3 = 0.2
+
+        @profile_function
+        def bar():
+            busy_loop(time1)
+
+        @profile_function
+        def foo():
+            busy_loop(time2)
+            bar()
+
+        @profile_function
+        def baz():
+            busy_loop(time3)
+            foo()
+
+        modmesh.call_profiler.reset()
+        bar()
+        bar()
+        foo()
+        baz()
+        root_stat = modmesh.call_profiler.stat()
+
+        # Check the number of lines
+        stat_line_list = root_stat.split("\n")
+        self.assertEqual(len(stat_line_list), 7)
+
+        # Check the first line
+        words = stat_line_list[0].split()
+        self.assertEqual(words[0], "7")
+        self.assertEqual(words[1], "function")
+        self.assertEqual(words[2], "calls")
+        self.assertEqual(words[3], "in")
+        ref_total_time = time1 * 4 + time2 * 2 + time3
+        self.assertTrue(abs(float(words[4]) - ref_total_time) <= 6e-4)
```

Comment:
Use [`assertLessEqual()`](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertLessEqual) and alikes.